test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,229 @@
+
+    We advise to use the Ubuntu Amazon EC2 AMI Locator to acquire the AMI value for a specific region: https://cloud-images.ubuntu.com/locator/ec2/. Select Ubuntu release `22.04` or `24.04`.
+
+    Also, use the var `EC2_SSH_PUBLIC_KEY` to specify the name of the uploaded public ssh key to your AWS account in the target region.
+
+    Upon completion of the script's execution, an Ansible inventory file containing the IP addresses of the master and worker nodes will be generated in the present directory at `.data/hosts_core`.
+
+2. Deploy a Kubernetes cluster:
+
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy-masters.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ansible-playbook -i .data/hosts_core deploy-workers.yaml --ssh-common-args='-o StrictHostKeyChecking=no'
+    ```
+
+3. Deploy KubeStellar in the hosting cluster:
+
+    Set var `KS_RELEASE` to release 0.25.1 or later. Then deploy KubeStellar: 
+    ```bash
+    ansible-playbook -i .data/hosts_core deploy_ks_core.yaml --ssh-common-args="-o StrictHostKeyChecking=no" -e "region=$REGION vpc_name=$VPC cluster_name=$CLUSTER_NAME ks_release=$KS_RELEASE"
+    ```
+
+    You can use the variable `ks_release` to specify the KubeStellar release. Kubestellar is deployed using the [KS helmchart](https://github.com/kubestellar/kubestellar/tree/release-0.26.0/core-chart) configured with a ITS of type host. 
+
+    Upon completion of the script's execution, a kubeconfig file is generated at `.data/admin.conf` to access the Kubernetes cluster and the control plane components for KubeStellar, for example:
+    
+    Check the created kubeconfig contexts:
+
+    ```bash 
+    kubectl --kubeconfig=$PWD/.data/admin.conf config get-contexts
+    ```
+
+    Output:
+    ```console
+    CURRENT   NAME     CLUSTER        AUTHINFO           NAMESPACE
+    *         kscore   kubernetes     kubernetes-admin   
+              wds1     wds1-cluster   wds1-admin         default
+    ```
+
+
+    To access the `wds1` k8s context first extract the command displayed at the end of the last Ansible task to edit your `etc/hosts` file with wds1 host entry:
+
+    (*Sample Ansible task command line output*):
+
+    ```console
+    TASK [Command to edit your etc/hosts file with wds1 host entry] ***********************************************************************
+    ok: [master1] => {
+        "msg": "echo '192.168.56.20 wds1.ec2-192-168-56-20.compute-1.amazonaws.com' | sudo tee -a /etc/hosts"
```

Comment:
This is unnecessarily complex. When I tested commit b00fdae41b4883c9af6c171759c855153fb6fe08, the IP address on this line was a global IP address (in net 13), the public IP address for my master node. Also, public DNS resolves the domain name to that IP address, if you just cut off that leading `wds1.` from the domain name.

Reply:
Oh, wait, it is not actually usable.

```console
bash-3.2$ kubectl --kubeconfig .data/admin.conf --context wds1 get ns
E0314 02:17:42.223627    3485 memcache.go:265] "Unhandled Error" err="couldn't get current server API group list: Get \"https://ec2-W-X-Y-Z.us-east-2.compute.amazonaws.com:32147/api?timeout=32s\": tls: failed to verify certificate: x509: certificate is valid for ingress.local, not ec2-W-X-Y-Z.us-east-2.compute.amazonaws.com"
...
```